UnityProject/Assets/Scripts/MapSaver/TestMapSaverScript.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using Newtonsoft.Json;
+using TileManagement;
+using UnityEngine;
+
+public class TestMapSaverScript : MonoBehaviour
+{
+	public MetaTileMap MetaTileMap;
+
+
+	[NaughtyAttributes.Button()]
+	public void SaveMap()
+	{
+		Logger.Log(JsonConvert.SerializeObject(MapSaver.SaveTileMap(MetaTileMap)));
```

Comment:
loggin the whole map is intended or was it for debugging?

Reply:
is debugging